util/logutil/slow_query_logger.go

```diff
@@ -32,6 +32,10 @@ func newSlowQueryLogger(cfg *LogConfig) (*zap.Logger, *log.ZapProperties, error)
 	// copy global config and override slow query log file
 	// if slow query log filename is empty, slow query log will behave the same as global log
 	sqConfig := cfg.Config
+	// enable/disable of slow logger is controlled by different configs, not the level of global logger
+	// the level of slow logger should always be "debug"
+	sqConfig.Level = zap.DebugLevel.String()
```

Comment:
It's meaningless to use `Debug` to log slow query here https://github.com/pingcap/tidb/blob/376d5428cca95ae51143162ee0f7f3972e84cff7/executor/adapter.go#L1028-L1029
because it should be inevitable to log it as long as force is true which checks the log level.
https://github.com/pingcap/tidb/blob/376d5428cca95ae51143162ee0f7f3972e84cff7/executor/adapter.go#L944